howto-conduct-a-privacy-review.md

```diff
@@ -113,8 +113,13 @@ of a privacy review, or might even file contradictory privacy issues if
 PING members disagree. Disagreement is rare. In the case that a PING privacy review results in 
 contradictory (or mutually exclusive) issues being filed, its the responsibility
 of the working group making the proposal (and, ultimately the director)
-to figure out how to proceed. However, the chairs will work with the working group and reviewer as needed with the aim of resolving any outstanding issues. 
+to figure out how to proceed. However, the chairs will work with the working group and reviewer
+as needed with the aim of resolving any outstanding issues.
 
+If a reviewer is unable to resolve an issue with the working group
+and the issue may lead to a formal objection, the reviewer
+is encouraged to disucss the issue with the PING chairs, and on a PING call.
```

Comment:
I might frame this as (to make it clear that PING will discuss it, not just the chairs):

> the reviewer is encouraged to bring the issue to the PING chairs, who can schedule discussion with PING on a call.